protostar/migrator/maybe_tmp_directory.py

```diff
@@ -0,0 +1,28 @@
+from contextlib import contextmanager
```

Comment:
file name does not match function name

Reply:
If the file name matches the function name, it:
- breaks patching
- autoimport imports a module rather than a function (at least in VS Code)

Reply:
🤨